src/cascadia/LocalTests_TerminalApp/TestHostApp/Package.appxmanifest

```diff
@@ -22,6 +22,13 @@
         </uap:DefaultTile>
         <uap:SplashScreen Image="taef.png" />
       </uap:VisualElements>
+      <Extensions>

```

Comment:
Don't we need to add this to `CascadiaPackage/Package-Pre.appxmanifest` as well?

I'm also guessing that adding this to the Preview appxmanifest won't interfere with the non-preview one right? (don't know much about extensions haha)

Reply:
Yes we do! added, thank you